packages/e2e-tests/specs/editor/blocks/gallery.test.js

```diff
@@ -69,6 +69,8 @@ describe( 'Gallery', () => {
 		expect( mediaLibraryButtonText ).toBe( 'Create a new gallery' );
 
 		// Unfortunately the Media Library has invalid HTML.
+		// Axe tests fail with the following error:
+		// `List element has direct children with a role that is not allowed: checkbox.`
```

Comment:
I haven't been able to track this down in the WordPress core codebase, but I'll keep looking.

Reply:
This was actually hidden in plain sight. I was looking for an `input type="checkbox"` but the `li` elements in the media modal all have `role="checkbox"` specified.

I made a trac ticket:
https://core.trac.wordpress.org/ticket/50273